<launch>
  <arg name="yaml" default="single_azure_mpaae"/>

  <rosparam command="load" file="$(find assembly_part_recognition)/params/$(arg yaml).yaml" ns="part_recognition" />
  <node name="furniture_pose_estimator" pkg="assembly_part_recognition" type="mpaae_server.py" output="screen"/>
</launch>